Jerrod/Assignments/puzzles.rb

```diff
@@ -0,0 +1,59 @@
+
+def firstPuzzle(arr)
+  sum = 0
+  arr.each{|i| sum += i}
+  puts sum
+  return arr.find_all{|i| i > 10}
+end
+puts firstPuzzle([3, 5, 1, 2, 7, 9, 8, 13, 25, 32])
+
+# Second Puzzle*************************
+def second(arr)
+  arr.shuffle.each{|i| puts i}
+  puts 'SPACE ************' #so i can tell the difference in the terminal
+  return arr.find_all{|i| i.length > 5}
+end
+puts second(['John', 'KB', "Oliver", "Cory", 'Matthew', 'Christopher'])
+
+# Third Puzzle*************************
+def third(arr)
+  puts arr.shuffle.at(arr.length-1)
+  x = arr.shuffle.at(0)
+  if x == 'a' || x == 'e' || x == 'i'|| x == 'o'|| x == 'u'
```

Comment:
This long if check works just fine.  An alternative could be to create an array with all of the vowels and then check if x is in the array:

vowels = ['a', 'e', 'i', 'o', 'u']
if vowels.include? x
  puts "${x} is a vowel"
else
  puts x
end